nomad/local/grapl-local-infra.nomad

```diff
@@ -270,7 +318,132 @@ job "grapl-local-infra" {
           }
         }
       }
+    }
+
+    task "kafka1" {
```

Comment:
seeing how much is repeated here I'd really prefer if we used a `dynamic "task" {}` which is literally perfect for these purposes. it iterates over the aforementioned `variable "kafka_broker_args"` and you can inject `i.broker_port`, `i.jmx_port` etc.

Reply:
see `    for_each = local.database_descriptors` for an example

Reply:
done